sites/dotnet-guide/src/pages/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import HomepageLayout from 'gatsby-theme-bulmaio/src/components/layout/HomepageLayout';
+// @ts-ignore
+import splash from './splash.svg';
+import { RecentItems } from 'gatsby-theme-bulmaio/src/components/recentitems/RecentItems';
+
+const dataUri = `url("${splash}") center center`;
+
+const IndexPage: React.FC = () => {
+
+  return (
+    <HomepageLayout title={`Home Page`}>
+      {{
+        hero: (
+          <section className="hero is-medium"
+                   style={{ background: dataUri, backgroundRepeat: 'no-repeat', backgroundSize: 1500 }}>
+            <div className="hero-body">
+              <div className="container">
+                <h1 className="title">.NET Tools Guide</h1>
+                <h2 className="subtitle">Learning resources for ReSharper Ultimate and Rider</h2>
+              </div>
+            </div>
+          </section>
+        ),
+        main: (
+          <RecentItems/>
```

Comment:
You don't have to do the Recent Items thing. If you'd prefer a "section" box that is just explanation, or other section boxes, I can point you in the right direction.

Reply:
Very happy to have it :)